client/src/components/graphs/IqrBoxPlot.js

```diff
@@ -0,0 +1,174 @@
+    d3.select(cursorRef.current).attr(
+      "transform",
+      `translate(${x} ${scaleYPosition})`
+    )
+  }, [x, scaleYPosition])
+
+  useEffect(() => {
+    const axis = d3.axisBottom(scale)
+    d3.select(axisRef.current).call(axis)
+  }, [scale])
+
+  const fillColor = d3.color("lightGray")
+  fillColor.opacity = 0.4
+
+  if (_isEmpty(values)) {
+    return whenUnspecified
+  }
+  return (
+    <svg
+      height={height}
+      width={width}
+      xmlns="http://www.w3.org/2000/svg"
+      ref={containerRef}
+    >
+      {values?.length > 1 && (
+        <g transform={`translate(0 ${scaleYPosition})`}>
+          <line
+            x1={scale(valuesStats.min)}
+            y1={0}
+            x2={scale(valuesStats.q1)}
+            y2={0}
+            style={{ stroke: "black", strokeWidth: 3 }}
+          />
+          <line
+            x1={scale(valuesStats.q1)}
+            y1={-15}
+            x2={scale(valuesStats.q1)}
+            y2={15}
+            style={{ stroke: "black", strokeWidth: 3 }}
+          />
+          <line
+            x1={scale(valuesStats.q2)}
+            y1={-15}
+            x2={scale(valuesStats.q2)}
+            y2={15}
+            style={{ stroke: "black", strokeWidth: 3 }}
+          />
+          <line
+            x1={scale(valuesStats.q3)}
+            y1={-15}
+            x2={scale(valuesStats.q3)}
+            y2={15}
+            style={{ stroke: "black", strokeWidth: 3 }}
+          />
+          <line
+            x1={scale(valuesStats.q1)}
+            y1={0}
+            x2={scale(valuesStats.q3)}
+            y2={0}
+            style={{ stroke: "black", strokeWidth: 3 }}
+          />
+          <line
+            x1={scale(valuesStats.q3)}
+            y1={0}
+            x2={scale(valuesStats.max)}
+            y2={0}
+            style={{ stroke: "black", strokeWidth: 3 }}
+          />
+        </g>
+      )}
+      <g ref={axisRef} transform={`translate(0 ${scaleYPosition})`} />
+
+      {onChange && (
+        <g ref={cursorRef}>
+          <polygon
+            points="0,0 13,13 13,30 -13,30 -13,13"
+            style={{
+              stroke: "gray",
+              fill: "",
+              strokeWidth: 1,
+              cursor: readonly ? null : "pointer"
+            }}
+          />
+        </g>
+      )}
+    </svg>
+  )
+}
+
+IqrBoxPlot.propTypes = {
+  values: PropTypes.arrayOf(PropTypes.number),
+  onChange: PropTypes.func,
+  levels: PropTypes.arrayOf(
+    PropTypes.shape({
+      color: PropTypes.string,
+      endValue: PropTypes.number.isRequired,
+      tooltip: PropTypes.string,
+      label: PropTypes.string
+    })
+  ).isRequired,
+  width: PropTypes.string.isRequired,
+  height: PropTypes.string.isRequired,
+  readonly: PropTypes.bool,
+  whenUnspecified: PropTypes.oneOfType([PropTypes.string, PropTypes.object])
+}
+
+IqrBoxPlot.defaultProps = {
+  levels: [
+    {
+      color: "red",
+      endValue: 3
+    },
+    {
+      color: "#FFBF00",
```

Comment:
I don't know. @VassilIordanov ?

Reply:
`#FFBF00` is Amber, as in RAG (Red, Amber, Green) - used in project management metaphors and traffic lights (at least in North America). In the Netherlands, traffic lights are Orange instead of Amber.

Reply:
It's unfortunate that CSS has no standard colour name for it, so a comment would be nice.

[Note: https://www.w3schools.com/w3css/w3css_colors.asp has amber, defined in CSS as:
```
.w3-amber,.w3-hover-amber:hover{color:#000!important;background-color:#ffc107!important}
.w3-text-amber,.w3-hover-text-amber:hover{color:#ffc107!important}
.w3-border-amber,.w3-hover-border-amber:hover{border-color:#ffc107!important}
```
]

Reply:
Fixed it in this commit: https://github.com/NCI-Agency/anet/pull/3069/commits/1415d9e5cd5454e854a24935ef9a767e0dea0cf7